starters/graphql-serverless-contentful/src/schema/technology/technology.resolver.test.ts

```diff
@@ -0,0 +1,197 @@
+import gql from 'graphql-tag';
+import { apolloServer } from '../../handlers/graphql';
+import { GraphQLResponse } from '@apollo/server/src/externalTypes';
+import assert from 'assert';
+import { Query } from '../../generated/graphql';
+import { client } from '../../utils/contentful';
+
+type TechnologyQuery = Pick<Query, 'technology'>;
+
+jest.mock('../../utils/contentful', () => ({
+	client: {},
+}));
+
+const MOCK_TECHNOLOGIES = [
+	{
+		id: '4f997875-a96d-4c84-ba43-39afd764168d',
+		displayName: 'GraphQL',
+		description:
+			'GraphQL provides a strong-typing system to better understand and utilize our API to retrieve and interact with our data.',
+		url: 'https://graphql.framework.dev/',
+	},
+	{
+		id: '2e6c7d25-8b03-4348-8329-cc4185225c71',
+		displayName: 'Node.js',
+		description:
+			'Node.js® is an open-source, cross-platform JavaScript runtime environment.',
+		url: 'https://nodejs.framework.dev/',
+	},
+];
+const MOCK_CONTENTFUL_ENTRIES = {
+	items: MOCK_TECHNOLOGIES.map(({ id, displayName, description, url }) => ({
+		sys: {
+			id,
+		},
+		fields: {
+			displayName: {
+				'en-US': displayName,
+			},
+			description: {
+				'en-US': description,
+			},
+			url: {
+				'en-US': url,
+			},
+		},
+		publish: () => undefined,
+		update: () => undefined,
+	})),
+};
+
+const getEntries = jest.fn(() => MOCK_CONTENTFUL_ENTRIES);
+const getEntry = jest.fn(() => MOCK_CONTENTFUL_ENTRIES.items[1]);
+const createEntry = jest.fn(() => MOCK_CONTENTFUL_ENTRIES.items[0]);
+// @ts-ignore
+client.getSpace = (key) =>
```

Comment:
You can add types for Space & Environment as done in #835, to avoid using @ts-ignore